src/kfmon.c

```diff
@@ -421,61 +419,61 @@ int nm_kfmon_list_request(const char *restrict foo __attribute__((unused))) {
 }
 
 // Giant ladder of fail
-nm_action_result_t* nm_kfmon_return_handler(int status, char **err_out) {
+nm_action_result_t* nm_kfmon_return_handler(kfmon_ipc_errno_e status, char **err_out) {
     #define NM_ERR_RET NULL
 
-    if (status != EXIT_SUCCESS) {
+    switch (status) {
+        case EXIT_SUCCESS:
```

Comment:
I think you meant `KFMON_IPC_OK`.

Reply:
Nnnnyeah, let's go with that for consistency's sake, because it's all 0 anyway, but nothing actually ever returns `KFMON_IPC_OK`, but does return `EXIT_SUCCESS` instead ;).